Assets/Scripts/Auction/AuctionStage.cs

```diff
@@ -22,111 +22,3 @@ public virtual bool Promote(out BiddingRound round)
         return true;
     }
 }
-
-[CreateAssetMenu(menuName = "Auction/Stage/Randomised")]
-public class RandomisedAuctionStage : AuctionStage
```

Comment:
I suppose we could add a new version of this back when refactoring is finished, if we find a need for easy in-editor testing of the bidding system.

Reply:
The script still exists, albeit in it's own file. This is due to Unity not accepting scriptableObject classes as arguments if their class name is different from their Class-file name. 🤪